### ISSUE TITLE
Archive this repository

### ISSUE THREAD

**Issue by newzealandpaul** [community-scripts/Proxmox#6]
Now that the frontend is merged into ProxmoxVE, should this repo be archived? 

@BramSuurdje

**Comment by Neustradamus** [comment]
@tteck: Why the repository https://github.com/tteck/Proxmox has not been moved into @community-scripts organization?

**Comment by BramSuurdje** [comment]
we first need to move all the traffic from /Proxmox to /ProxmoxVE. than we can delete this repo all together

**Comment by Neustradamus** [comment]
@BramSuurdje: No, with a transfer of a repository, redirection will be here, no lost.

Steps:
- First, unarchive and transfer https://github.com/tteck/Proxmox to https://github.com/community-scripts/Proxmox
- Second, unarchive and update the current https://github.com/BramSuurdje/proxmox-helper-scripts repository with all from the fork https://github.com/community-scripts/Proxmox
- Remove current https://github.com/community-scripts/Proxmox (after pushed the code into https://github.com/BramSuurdje/proxmox-helper-scripts)
- Transfer https://github.com/BramSuurdje/proxmox-helper-scripts to https://github.com/community-scripts

**Comment by Neustradamus** [comment]
@BramSuurdje: No, with a transfer of a repository, redirection will be here, no lost.

Steps:
- @tteck: First, unarchive and transfer https://github.com/tteck/Proxmox to https://github.com/community-scripts/Proxmox
- @BramSuurdje: Second, unarchive and update the current https://github.com/BramSuurdje/proxmox-helper-scripts repository with all from the fork https://github.com/community-scripts/Proxmox
- Remove current https://github.com/community-scripts/Proxmox (after pushed the code into https://github.com/BramSuurdje/proxmox-helper-scripts)
- Transfer https://github.com/BramSuurdje/proxmox-helper-scripts to https://github.com/community-scripts

@BramSuurdje: If you need help, I can do it, please add me right on https://github.com/BramSuurdje/proxmox-helper-scripts and ownership on https://github.com/community-scripts for the best.

Thanks in advance.

**Comment by Neustradamus** [comment]
@BramSuurdje: Any progress?
